doc/content/source/vectorpostprocessors/ResponseHistoryMean.md

```diff
@@ -0,0 +1,26 @@
+# ResponseHistoryMean
+
+`ResponseHistoryMean` computes the mean history given the response histories of a suite of nodes. These nodal response histories are computed using the `ResponseHistoryBuilder` class referenced as a vector of pointers. This vector is then passed on to `ResponseHistoryMean` to compute the mean history calculated as a function of time and will be stored in a csv file.
+
+## Usage
```

Comment:
Does this look good?

_**`ResponseHistoryBuilder` has the capability to calculate response quantities of different types (i.e., acceleration_x, acceleration_y, velocity_x, displacement_y) at a list of nodes or a boundary. If the `ResponseHistoryBuilder` VectorPostprocessor requesting response histories of different types is input into `ResponseHistoryMean`, then mean across all these types will be calculated by `ResponseHistoryMean`. Of course, this may not be the intent of the user. It is, therefore, the responsibility of the user to ensure that `ResponseHistoryBuilder` has only one response quantity whose histories are recorded at a list of nodes or a boundary. `ResponseHistoryMean` will not perform a check to ensure this.**_